assets/locale/el/messages.po

```diff
@@ -1955,6 +1959,34 @@ msgstr "Όνομα"
 msgid "py.forms.signup.lastName.placeholder"
 msgstr "Επώνυμο"
 
+#: fetcher/fetch.py:187
+msgid "py.fetcher.fetch.user_visible_bug_fetch_result"
+msgstr ""
+
+#: fetcher/fetch.py:289
+msgid "py.fetcher.fetch.fetch_or_wrap_error.no_loaded_module"
```

Comment:
Nix `fetch_or_wrap_error` from the message ID, please. Just `py.fetcher.fetch.no_loaded_module` is fine. `fetch_or_wrap_error` ought to be a private function; it shouldn't appear in our pofiles. Same goes for subsequent messages)